CI/conan/mingw-linux

```diff
@@ -0,0 +1,36 @@
+toolchain=/usr/i686-w64-mingw32
+target_host=i686-w64-mingw32
+
+[settings]
+
+# We are cross-building to Windows
+os=Windows
+arch=x86
```

Comment:
Also - why x86?
It looks that our mxe build is 32-bit but don't really see any reason to keep using it.

Reply:
MSVC build is 64-bit and seems to be stable, so no reason not to use 64-bit builds for Windows

Reply:
@IvanSavenko yes, 64bit is stable, but I think than 32-bit windows is more used in CIS, especially in old PCs (with Windows XP or Windows 7 onboard). So, what if MSVC will remain 64bit, and Conan can be both 32bit and 64 bit?

Reply:
Not really. Maybe some ancient embedded systems, but all players use x64:
https://store.steampowered.com/hwsurvey
0.11% Windows 7 (32-bit)

We also had plans to use msvc as primary build (and one we distribute as release), which is 64-bit.

Reply:
64-bit only sounds fine. If there's a request from community, we can make 32-bit ones as well.

Reply:
@IvanSavenko @kambala-decapitator okay, I will regenerate to 64-bit. And I will add strip flag to CPACK for Release builds. It is okay?

Reply:
OK.
Strip flag - yes, for "Release" build, keep debug info for "RelWithDebugInfo" or what's its name

Reply:
sounds good. let's still keep 32-bit deps in GitHub release in case someone wants to make a local 32-bit build with mingw.

Reply:
Done both strip and 64bit.